app/helpers/application_helper.rb

```diff
@@ -12,7 +12,7 @@ def published_document_path(document)
   end
 
   def state(document)
-    state = document.publication_state
+    state = document.publication_state == "live" ? "published" : document.publication_state
```

Comment:
@benilovj for this change to work we will need to have:

```
  unless document.publication_state.nil?
      state = document.publication_state.gsub("live", "published")
    end
```

Otherwise this test will fail when a content_id does not exist

```
  scenario "that doesn't exist" do
    content_id = "a-case-that-doesnt-exist"
    publishing_api_does_not_have_item(content_id)

    visit "/cma-cases/#{content_id}"

    expect(page.current_path).to eq("/cma-cases")
    expect(page).to have_content("Document not found")
  end
```

Do you prefer this to how it was previously?